h2o-py/h2o/model/model_base.py

```diff
@@ -962,6 +962,26 @@ def mean_residual_deviance(self, train=False, valid=False, xval=False):
         for k, v in viewitems(tm): m[k] = None if v is None else v.mean_residual_deviance()
         return list(m.values())[0] if len(m) == 1 else m
 
+    def mean_residual_deviance_not_optimized(self, train=False, valid=False, xval=False):
+        """
+        Get the Mean Residual Deviances.
+
```

Comment:
@syzonyuliia This is not exactly the same mean residual deviance for Poisson family?  I thought you meant that for all other families, it is the same as the mean residual deviance.  For for Poisson, it is different because it uses a different formula, right?